Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -125,8 +126,9 @@ class TextureCacheBase
   virtual void DeleteShaders() = 0;
 
   TCacheEntry* Load(const u32 stage);
-  void UnbindTextures();
-  virtual void BindTextures();
+  static void InvalidateAllBindPoints() { invalidated_binds = 0xff; }
+  static bool IsValidBind(u8 i) { return (invalidated_binds & (1 << i)) == 0; }
+  virtual void BindTexture(u8 i);
```

Comment:
I removed the virtual there, i hope that's correct, not an expert on these details